docs/source/share_kedro_viz.md

```diff
@@ -1,136 +1,46 @@
-# Publish and share Kedro-Viz
-
-```{note}
-Kedro-Viz sharing was introduced in version 6.6.0.
-```
-
-This page describes how to publish Kedro-Viz to a static website hosting platform to share it with others. It uses the {doc}`spaceflights tutorial<kedro:tutorial/spaceflights_tutorial>` as an example.
-
-If you haven't installed Kedro {doc}`follow the documentation to get set up<kedro:get_started/install>`. 
-
-```{important}
-We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.19.0). To check the version installed, type `kedro -V` in your terminal window.
-```
-
-In your terminal window, navigate to the folder you want to store the project. Generate the spaceflights tutorial project with all the code in place by using the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas):
-
-
-```bash
-kedro new --starter=spaceflights-pandas
-```
-
-When prompted for a project name, you can enter anything, but we will assume `Spaceflights` throughout.
-
-When your project is ready, navigate to the root directory of the project.
+# Publish and Share Kedro-Viz
 
+Kedro users want to share their pipeline visualisation with other non-technical  stakeholders on their team. 
 
-## Publish and share Kedro-Viz automatically
+Kedro enables users to do this by publishing Kedro-Viz to a static website hosting platform to share it with others. This guide uses the spaceflights tutorial as an example.
 
+There are two ways to publish and share Kedro-Viz:
```

Comment:
I found it a bit confusing that this says there are two ways of sharing Viz and then below there's also the header "Publish and share via CLI", so is this a third way or is it part of one of the other two?